backend/inspirehep/hepdata/cli.py

```
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+from datetime import datetime, timedelta
+
+import click
+import requests
+from invenio_db import db
+
+from inspirehep.records.api import LiteratureRecord
+
+HEPDATA_URL = "https://www.hepdata.net/search/ids"
+
+
+@click.group()
+def hepdata():
+    """Commands for disambiguation"""
+
+
+@hepdata.command("harvest")
+@click.option(
+    "--since",
+    "-s",
+    "since",
+    default=None,
+    help="Date from when hepdata should be harvested. If not provided today's date would be used.",
+)
+def harvest(since):
+    if since:
+        try:
+            since = datetime.strptime(since, "%Y-%m-%d").date()
+        except ValueError:
+            raise ValueError(
+                f"`since`: {since} is in wrong format. Should be in ISO format: YYYY-MM-DD."
+            )
+    else:
+        since = (datetime.now() - timedelta(1)).strftime("%Y-%m-%d")
+    payload = {"inspire_ids": True, "last_updated": since, "sort_by": "latest"}
+    hepdata_response = requests.get(HEPDATA_URL, params=payload)
+    try:
+        hepdata_response.raise_for_status()
+    except Exception as exc:
+        message = "Hepdata harvest failed!"
+        click.echo(click.style(message, fg="red"))
+        click.echo(exc.message)
```

what's the point of this? I think you can just let it fail and the exception will go to sentry and the logs.